CHANGES_NEXT_RELEASE

```diff
@@ -1,5 +1,6 @@
+Add trimestamp in device and group provision (#655)
```

Comment:
trimestamp -> timestamp

Reply:
fixed in e47193061f69165511c7d854f1832d93730b1b24